kalite/distributed/management/commands/setup.py

```diff
@@ -455,6 +455,7 @@ def handle(self, *args, **options):
         # Now deploy the static files
         logging.info("Copying static media...")
         call_command("collectstatic", interactive=False, verbosity=0)
+        call_command("compress", interactive=False, verbosity=0)
```

Comment:
It has internally bundled libraries to do all the compressing by default, to use more efficient/'better' compilers you can set it to defer to external libraries.